frappe/core/doctype/test/test.json

```diff
@@ -36,7 +36,9 @@
    "write": 1
   }
  ],
+ "read_only": 1,
  "sort_field": "modified",
  "sort_order": "DESC",
+ "states": [],
```

Comment:
Should we rename this to "Test Virtual DocType" instead? As "test", it looks like yet another directory that has been tracked as a blunder. 😆

Reply:
Ideally, this should not be in the core doctype. I guess it was committed unknowingly.